engine/src/main/java/org/terasology/engine/subsystem/headless/renderer/HeadlessWorldRenderer.java

```diff
@@ -192,6 +193,11 @@ public String getMetrics() {
         return "";
     }
 
+    @Override
```

Comment:
I decided to go with the easier option for now - exposing the RenderGraph via WorldRenderer.